config/confighttp/confighttp.go

```diff
@@ -281,6 +281,13 @@ type ServerConfig struct {
 	ResponseHeaders map[string]configopaque.String `mapstructure:"response_headers"`
 }
 
+// NewDefaultServerConfig returns ServerConfig type object with default values.
+// Currently, config options are all initialized as zero values.
+// We encourage to use this function to create an object of ServerConfig.
+func NewDefaultServerConfig() ServerConfig {
+	return ServerConfig{}
```

Comment:
Can I suggest setting the write timeout here already on this first implementation? This way, once we set that as the default, consumers of this new function won't be affected.

Reply:
Sure thing. What value?

Reply:
Let's pick a number: 30s